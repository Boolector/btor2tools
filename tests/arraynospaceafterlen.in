1 array 1nospace
