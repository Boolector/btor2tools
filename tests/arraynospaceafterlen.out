*** catbtor: parse error in 'arraynospaceafterlen.in' line 1: expected space after length
